### PR TITLE
Update standard.md

### PR DESCRIPTION
**Pull request by Not-Whiskey**
Made edits for grammar and consistency in capitalization. Added hyperlink for the Principles for Digital Development so as to align with other suggested resources made throughout which were hyperlinked.

### CODE REVIEW

File: standard.md [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
+**2. Use of Approved Open Licenses** | Projects must demonstrate the use of an approved open license. For open source software, only [OSI approved licenses](https://opensource.org/licenses) are accepted. For open content the use of a [Creative Commons license](https://creativecommons.org/licenses/) is required. While we encourage projects to use a license that allows for both derivatives and commercial reuse ([CC-BY](https://creativecommons.org/licenses/by/4.0/) and [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)), or dedicate content to the public domain ([CC0](https://creativecommons.org/choose/zero/)); licenses that do not allow for commercial reuse ([CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) and [CC-BY-NC-SA](https://creativecommons.org/licenses/by-nc-sa/4.0/)) are also accepted. For open data, an [Open Data Commons approved license](https://opendefinition.org/licenses/) is required. [The full license list is here for reference](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md).
```
Comment (lacabra) [inline standard.md:10]: To hyperlink this whole sentence `The full license list is here for reference` does not look quite right. Could you propose alternative wording, perhaps like:

See [the full license list](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md).

File: standard.md [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
+**4. Platform Independence** | If the project has mandatory dependencies that create more restrictions than the original license, the project(s) must be able to demonstrate independence from the closed component(s) and/or indicate the existence of functional, open alternatives.
+**5. Documentation** | The project must have documentation of the source code, use cases, and/or functional requirements. For content, this should include all relevant/compatible apps, software, or hardware required to access the content, and instructions regarding how to use it. For software projects, this should be technical documentation that would allow a technical person unfamiliar with the project to launch and run the software. For data projects, this should be documentation that describes all the fields in the set, and provides context on how data was collected, and how it should be interpreted.
+**6. Mechanism for Extracting Data** | If the project has non personally identifiable information (PII) there must be a mechanism for extracting or importing non PII-data from the system in a non-proprietary format.
```
Comment (lacabra) [inline standard.md:14]: the hyphen in `non PII-data` seems off-placed. I believe it should read `non-PII data`

File: standard.md [hunk 3]
```diff
@@ -4,19 +4,19 @@
...
+**7. Adherence to Privacy and Applicable Laws** | The project must state to the best of its knowledge that it complies with relevant privacy laws, and all applicable international and domestic laws.
+**8. Adherence to Standards & Best Practices** | Projects must demonstrate adherence to standards, best practices, and/or principles. For example, [the Principles for Digital Development](https://digitalprinciples.org/principles/).
+**9. Do No Harm** | All projects must demonstrate that they have taken steps to ensure the project anticipates, prevents, and does no harm.
+**9a) Data Privacy & Security** | Projects that collect data must identify the types of data collected and stored and demonstrate that the project ensures the privacy and security of this data and has taken steps to prevent adverse impacts resulting from it’s collection, storage and distribution.
```
Comment (lacabra) [inline standard.md:18]: This is an awfully long sentence with 5 `and` clauses. Kindly break it down in shorter sentences.

**Review by lacabra — CHANGES_REQUESTED** [review]
Thanks @Not-Whiskey for your contribution and congratulations on your first Pull Request 💪

I have noticed some additional areas that need attention and I have flagged above. Could you please address them before I can approve and merge it?

Thanks again 🙏

File: standard.md (hunk 1 above)
Comment (Not-Whiskey) [reply standard.md:10]: Change made!

File: standard.md (hunk 2 above)
Comment (Not-Whiskey) [reply standard.md:14]: Thank you!

File: standard.md (hunk 3 above)
Comment (Not-Whiskey) [reply standard.md:18]: Agreed - just changed to make it easier to follow/read.

**Review by lacabra — APPROVED** [review]
Fantastic! Thanks for addressing my requests for change so promptly!
This looks good to me; thanks @Not-Whiskey !

**Pull request merged**